site-src/geps/gep-1426.md

```diff
@@ -146,6 +141,23 @@ This should be enforced by controller validation, which should set an `Accepted:
 
 The logic described here is a bit verbose, but could be validated with conformance tests, and is intended to allow an `HTTPRoute` configuring mesh traffic rules to additionally specify a `Gateway` `parentRef` (or `HTTPRoute` `parentRef` if [GEP-1058: Route Inclusion and Delegation](https://github.com/kubernetes-sigs/gateway-api/pull/1085) moves forward) to apply the same routing rules when exposing a mesh service outside the cluster.
 
+### Namespace boundaries
+
+In a mesh, routes can be configured by two personas:
+
+* Service producers, who want to modify behavior of *inbound* requests to their services
```

Comment:
Suggestion: unless this term is explicitly defined somewhere, “providers” might be clearer than “producers”. Or maybe “request receivers” and “request makers”? Not sure if that’s much better but something about “service producers” feels a little off in this context.

Reply:
Maybe I can add a link

Reply:
The service producer/consumer roles are described in https://gateway-api.sigs.k8s.io/geps/gep-1324/#producer-and-consumer, I can open a separate PR linking that in the Personas section of this GEP.